puphpet/config.yaml.template

```diff
@@ -208,13 +264,20 @@ python:
 mysql:
     install: '1'
     override_options: {  }
-    root_password: '123'
+    root_password: password
```

Comment:
Not inane. No self-deprecation is necessary here. When I argue passionately for something, it's the most I can ask for for someone else to do just as much in favor of their position. A solid counter-argument is the only way to prove whether one's own argument has merit or flaws and to refine it appropriately based on that feedback. Thank you for not giving up.